docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+version: '3.8'  
+
+services:
+  postgres: 
+    image: postgres:latest 
+    container_name: postgres_db
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres      
+      POSTGRES_PASSWORD: password  
+      POSTGRES_DB: maxitdb          
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    networks:
+      - app_network
+
+  rabbitmq:
+    image: rabbitmq:3.13-management
+    container_name: rabbitmq
+    ports:
+      - "5672:5672"
+      - "15672:15672"
```

Comment:
Why do we expose 2 ports?

Reply:
port 5672 for client connections 
port 15672 for Management UI - in a testing phase it is needed to actually send messages to the queue later can be deleted.